protostar/starknet_gateway/gateway_facade_input_validation.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
Does this file need to be prefixed with `gateway_facade`?  Maybe `input_validators.py` would be descriptive enough.